practice-three/src/js/script.js

```diff
@@ -178,34 +178,29 @@ const renderUserTable = () => {
 /**
  * Function POST data to userDatabase 
  */
- const createUsers = (data, callback) => {
+ const createUser = (data) => {
   const options = {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json'
     },
     body: JSON.stringify(data)
   };
-  fetch(courseApi, options)
+
+  fetch(userApi, options)
```

Comment:
No need to handle error of this API call?

Reply:
fixed